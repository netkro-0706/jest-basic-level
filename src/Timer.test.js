import { render } from "@testing-library/react";
import Timer from "./Timer";

//지속적으로 변경되는 데이터 테스트
//mock함수를 만들어서 대응
test("초 표시", () => {
  Date.now = jest.fn(() => 123456789);
  const { container } = render(<Timer />);
  expect(container).toMatchSnapshot();
});
