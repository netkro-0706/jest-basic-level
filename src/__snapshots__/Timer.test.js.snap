// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`초 표시 1`] = `
<div>
  <p>
    현재 
    36
    초 입니다.
  </p>
</div>
`;
